guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>Comment: The topics needs to be sequenced in order to make it coherent. </p><p> </p><p>&nbsp;Its architecture can be broadly thought of as an&nbsp;<strong>encoder</strong>&nbsp;network followed by a&nbsp;<strong>decoder</strong>&nbsp;network.&nbsp;</p><p>Explain PSPNet as encoder. </p><ul><li>Dilated convolution</li><li>Pyramid pooling module</li></ul><p>Explaing PSPNet as decoder </p><ul><li>final predictions</li></ul><p>Before we begin explaining about dilated convolutions, we should show the image that comes in this portion. Then explaining (a), ..(d) parts one by one will make the concept more clear.</p><blockquote>The PSPNet model is not a complete segmentation model in itself, it is just an encoder, which means it is just half of what is required for image segmentation. The most common decoders that are found in various implementations is a convolution layer followed by a 8x bilinear-upsampling.</blockquote><blockquote><a href="https://github.com/hszhao/PSPNet/issues/101" target="_blank">Source: Github PSPNet</a></blockquote><blockquote>There is a downside of having a 8x upsmapling in the end is that there is no learnable layers hence the results that we get are blobs as results and fails to capture high resolution detailed information.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Restructured the notebook according to the suggestions.</p>